torchao/prototype/common/bitpacking.py

```diff
@@ -27,7 +28,9 @@ def unpack(data: torch.Tensor,
     """
     container_size = torch.iinfo(data.dtype).bits
     scale = container_size // element_bit_width
-    
+    if device is None:
```

Comment:
there is no `device` before this point. Don't need to check if device is None, just use `device = data.device`